recipes/pyoptsparse/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "pyoptsparse" %}
+{% set version = "2.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://github.com/mdolab/pyoptsparse/archive/refs/tags/v{{ version }}.tar.gz"
+  sha256: 4f28eefa4eb5724f3d0a2b6118c45aedee24a7dea72a6672a49feb77220ecf32
+  patches:
+    - patch-IPOPT-setup-for-conda.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - ipopt 3.13.*
```

Comment:
Remove this. This should only be a `host` dep

Reply:
I was unable to get it to build (locally) with this as a host dep. I will push to see if this is reproducible on CI...

Reply:
Same result on CI. Here is a link to error:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=336240&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=3124